simpl/tests/test_simpl.py

```diff
@@ -43,6 +43,7 @@ def test_anon(self):
         self.assertEqual(json.loads(request.content)["data"], {"runs": None})
 
     def test_auth(self):
+        Run.objects.all().delete()
```

Comment:
Hmm. If we need to delete DB objects at the start of a test that implies there is some cleanup that is not happening somewhere in the test suite.

Can we see if we can find that and fix the root cause rather than adding a band-aid here.